src/main/java/uk/nhs/digital/nhsconnect/nhais/translator/removal/RemovalTranslator.java

```diff
@@ -31,9 +32,13 @@
     private final GpNameAndAddressMapper gpNameAndAddressMapper;
     private final PersonNameMapper personNameMapper;
     private final FreeTextMapper freeTextMapper;
+    private final OptionalInputValidator validator;
 
     @Override
     public List<Segment> translate(Parameters parameters) throws FhirValidationException {
+        if(validator.nhsNumberIsMissing(parameters)) {
+            throw new FhirValidationException("Patient resource property /identifier/0/value (NHS Number) is required");
```

Comment:
To have a proper JSON path use our const from JsonPatch

Reply:
I need to add a constant to the JsonPatches then